starport/templates/typed/stargate/proto/{{moduleName}}/v1beta/{{typeName}}.proto.plush

```diff
@@ -5,9 +5,25 @@ option go_package = "<%= ModulePath %>/x/<%= ModuleName %>/types";
 
 import "gogoproto/gogo.proto";
 
-message Msg<%= title(TypeName) %> {
+message <%= title(TypeName) %> {
   string id = 1;
   bytes creator = 2 [(gogoproto.casttype) = "github.com/cosmos/cosmos-sdk/types.AccAddress"];<%= for (i, field) in Fields { %>
   <%= field.Datatype %> <%= field.Name %> = <%= i+3 %>; <% } %>
 }
 
+message MsgCreate<%= title(TypeName) %> {
+  string id = 1;
```

Comment:
It's the `id` created when the type is created?
Should be removed then

Reply:
solved by 4d69a84